deps/wazuh_testing/wazuh_testing/modules/vulnerability_detector/event_monitor.py

```diff
@@ -295,3 +295,12 @@ def check_refresh_feed_log(provider_name):
     check_vuln_detector_event(timeout=vd.VULN_DETECTOR_GLOBAL_TIMEOUT,
                               callback=f"Refresh of '{provider_name}' database finished",
                               error_message=f"Could not find {provider_name} feed refresh log")
+
+
+def check_configuration_error():
+    """Check the configuration error event in ossec.log"""
+    check_vuln_detector_event(timeout=vd.DOWNLOAD_TIMEOUT,
+                              callback=r".* \(\d+\): Configuration error at.*",
+                              error_message=f"Could not find the event 'Configuration error at "
+                                            f"'etc/ossec.conf' in ossec.log",
```

Comment:
`fstring` is not necessary

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/a6bca2e3641a2ab812ce3de8bfa09045b5fcd4e9